src/spikeinterface/sorters/launcher.py

```diff
@@ -68,10 +67,18 @@ def run_sorter_jobs(job_list, engine="loop", engine_kwargs={}, return_output=Fal
         The engine to run the list.
         * "loop" : a simple loop. This engine is
     engine_kwargs : dict
-
-    return_output : bool, dfault False
+        In the case of engine="slum", possible kwargs are:
+        - tmp_script_folder: str, default None
+            the folder in which the job scripts are created. Default: directory created by
+            the `tempfile` library
+        - sbatch_executable_path: str, default 'sbatch'
+            the path to the `sbatch` executable
+        - other kwargs are interpreted as arguments to sbatch, and are translated to the --args to be passed to sbatch.
+          see the [documentation for `sbatch`](https://slurm.schedmd.com/sbatch.html) for a list of possible arguments
+
+    return_output : bool, default False
```

Comment:
```suggestion
    return_output : bool, default: False
```

Reply:
Happy to follow whatever guidelines you recommend. The advantage of pop-ing out the few arguments that we want to use for something else is to not have to specify which aruments to `sbatch` we accept. That way, if `sbatch` were to add an additional argument, for instance, then we do not need to update spikeinterface to match.

Reply:
Since I don't use slurm I'm not sure. Some times there are json dump of some params for provenance, so I would wait to see whatever Sam and Alessio say regarding that.